CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -0,0 +1,85 @@
+# Changelog 1.14
+
+## 1.14.0 - 2023-10-27
+
+### Added
+
+- Add rate-limit config to Egress to specify the rate limit of north-south egress traffic of this Egress. ([#5425](https://github.com/antrea-io/antrea/pull/5425), [@GraysonWu])
+- Add `IPAllocated` and `IPAssigned` conditions to Egress status to improve Egress visibility. ([#5282](https://github.com/antrea-io/antrea/pull/5282), [@AJPL88] [@tnqn])
+- Add goroutine stack dump in `SupportBundle` for both Antrea Agent and Antrea Controller. ([#5538](https://github.com/antrea-io/antrea/pull/5538), [@aniketraj1947])
+- Add "X-Load-Balancing-Endpoint-Weight" header to AntreaProxy Service healthcheck. ([#5299](https://github.com/antrea-io/antrea/pull/5299), [@hongliangl])
+- Add log rotation configuration in Antrea Agent config for audit logs. ([#5337](https://github.com/antrea-io/antrea/pull/5337) [#5366](https://github.com/antrea-io/antrea/pull/5366), [@antoninbas] [@mengdie-song])
+- Add GroupMembers API Pagination support to Antrea Go clientset. ([#5533](https://github.com/antrea-io/antrea/pull/5533), [@qiyueyao])
+- Add Namespaced Group Membership API for Antrea Controller. ([#5380](https://github.com/antrea-io/antrea/pull/5380), [@qiyueyao])
+- Support Pod secondary interfaces on VLAN network. ([#5341](https://github.com/antrea-io/antrea/pull/5341) [#5365](https://github.com/antrea-io/antrea/pull/5365) [#5279](https://github.com/antrea-io/antrea/pull/5279), [@jianjuns])
+- Add FlexibleIPAM support for Antrea Multicast. ([#4922](https://github.com/antrea-io/antrea/pull/4922), [@ceclinux])
+- Enable Windows OVS container to run on pristine host environment, without requiring some dependencies to be installed manually ahead of time. ([#5440](https://github.com/antrea-io/antrea/pull/5440), [@NamanAg30])
+- Update `Install-WindowsCNI-Containerd.ps1` script to make it compatible with containerd 1.7. ([#5528](https://github.com/antrea-io/antrea/pull/5528), [@NamanAg30])
+- Add a new all-in-one manifest for the Multi-cluster leader cluster, and update the Multi-cluster user guide. ([#5389](https://github.com/antrea-io/antrea/pull/5389) [#5531](https://github.com/antrea-io/antrea/pull/5531), [@luolanzone])
+- Clean up auto-generated resources in leader and member clusters when a ClusterSet is deleted, and recreate resources when a member cluster rejoins the ClusterSet. ([#5351](https://github.com/antrea-io/antrea/pull/5351) [#5410](https://github.com/antrea-io/antrea/pull/5410, [@luolanzone])
+
+### Changed
+
+- Multiple APIs are promoted from beta to GA. The corresponding feature gates are removed from Antrea config files.
+  - Promote feature gate EndpointSlice to GA. ([#5393](https://github.com/antrea-io/antrea/pull/5393), [@hongliangl])
+  - Promote feature gate NodePortLocal to GA. ([#5491](https://github.com/antrea-io/antrea/pull/5491), [@hjiajing])
+  - Promote feature gate AntreaProxy to GA. ([#5401](https://github.com/antrea-io/antrea/pull/5401), [@hongliangl])
```

Comment:
should mention the addition of `antreaProxy.enable` option.

Reply:
updated.